Makefile

```diff
@@ -116,6 +116,12 @@ safety: ## Runs `safety check` to check python dependencies for vulnerabilities
 		|| exit 1; \
 	done
 
+# Bandit is a static code analysis tool to detect security vulnerabilities in Python applications
+# https://wiki.openstack.org/wiki/Security/Projects/Bandit
+.PHONY: bandit
+bandit: ## Run bandit with medium level excluding test-related folders
+	@bandit --recursive . --exclude molecule,testinfra,securedrop/tests,admin/.tox -ll
```

Comment:
This is passing CI, _but_ additional dirs should be excluded so that devs can run the target locally:

* `admin/.venv/`
* `admin/.eggs/`
* `.venv/`

Those directories will be created during testing in Tails, e.g. using prod VMs.

Reply:
and `.tox`